.goreleaser.yml

```diff
@@ -39,7 +39,8 @@ builds:
         goarch: 386
 
 archives:
-  -
+  - name_template: >-
```

Comment:
Why remove the `-rc` from the filename? Isn't it better to make it clear it's a release candidate?

Reply:
Then we can't simply promote an `-rc` by creating a new tag, as we've been doing.

If we want to go the route where we keep the `-rc` in the filename, we'd have to either:
1. Delete the existing assets and create a new release by running `goreleaser` again.
2. Delete the existing pre-release and create a new release by running `goreleaser` again.
3. Somehow change the name of all the assets and checksum file. I'm not sure how easy that would be?

For 1 and 2 we run the risk of introducing (build environment) changes between the pre-release and the release that's published. Not sure how easy 3 is to automate, but it could be messy.

Reply:
I believe you are trying to simply upgrade the candidate binary. If that's what you want to do, can you confirm that the correct version (without the -rc) is being injected here: https://github.com/Jigsaw-Code/outline-ss-server/blob/d240aa19d895910a85bd5d5d2bfc92cae4e98396/cmd/outline-ss-server/main.go#L44. Otherwise you will need to figure out how to inject the version correctly, or rebuild like we used to.

For context, we didn't use to create a release for candidates, only set the tag. I think we created prereleases sometimes. Once confident about the candidate, the release was created. In that case you don't need all those steps you listed. `vN-rc-M` can be one prerelease and `vn` can be another. There was no need to edit releases.